deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -12,9 +12,45 @@ products:
 
 A [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
 
-ECK manages a default PDB per {{es}} resource. It allows one {{es}} Pod to be taken down, as long as the cluster has a `green` health. Single-node clusters are not considered highly available and can always be disrupted.
+{{eck}} manages either a single default PDB, or multiple PDBs per {{es}} resource depending on the license level of the ECK installation.
 
-In the {{es}} specification, you can change the default behavior as follows:
+:::{note}
+In {{eck}} 3.1 and earlier, all clusters follow the [default PodDisruptionBudget rules](#default-pdb-rules), regardless of license type.
+:::
+
+## Advanced PodDisruptionBudget rules (Enterprise License Required)
+```{applies_to}
+deployment:
+  eck: ga 3.2
+```
+
+In Elasticsearch clusters managed by ECK and licensed with an enterprise license, a separate PDB is created for each type of nodeSet defined in the manifest. This setup allows upgrade or maintenance operations to be executed more quickly. Each PDB permits one Elasticsearch Pod per nodeSet to be disrupted at a time, provided the cluster maintains the health status described in the following table:
```

Comment:
```suggestion
In Elasticsearch clusters managed by ECK and licensed with an Enterprise license, a separate PDB is created for each type of nodeSet defined in the manifest. This setup allows upgrade or maintenance operations to be executed more quickly. Each PDB permits one Elasticsearch Pod per nodeSet to be disrupted at a time, provided the cluster maintains the health status described in the following table:
```

It seems we capitalize the word "Enterprise" on our other pages e.g. https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2361/deploy-manage/license/manage-your-license-in-eck